src/workerd/api/encoding.c++

```diff
@@ -256,6 +257,7 @@ kj::StringPtr getEncodingId(Encoding encoding) {
 }
 
 Encoding getEncodingForLabel(kj::StringPtr label) {
+  kj::String labelInsensitive = toLower(label);
```

Comment:
This calls `kj::String toLower(kj::String&& str)` in util.h. Calling that function with a kj::StringPtr is safe and just results in the string being implicitly copied, right?

Reply:
Turns out that this is using the `jsg::ByteString toLower(kj::StringPtr str);` constructor in the same file instead. This feels wonky, but works correctly for this use case.